etc/config/c++.amazon.properties

```diff
@@ -416,6 +416,11 @@ compiler.clang1810.semver=18.1.0
 compiler.clang1810.options=--gcc-toolchain=/opt/compiler-explorer/gcc-13.2.0
 compiler.clang1810.ldPath=${exePath}/../lib|${exePath}/../lib/x86_64-unknown-linux-gnu
 compiler.clang1810.debugPatched=true
+compiler.clang1810_clad.exe=/opt/compiler-explorer/clang-18.1.0/bin/clang++
```

Comment:
This looks ok -- will get the compilers etc rebuilt and installed and test before we merge. thanks!